core/integration/features/verified/closeout-cascade.feature

```diff
@@ -37,7 +37,7 @@ Feature: Closeout-cascades
     When the parties submit the following liquidity provision:
       | id  | party  | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type    |
       | lp1 | lpprov | ETH/DEC19 | 100000            | 0.001 | sell | ASK              | 100        | 55     | submission |
-      | lp1 | party0 | ETH/DEC19 | 100000            | 0.001 | buy  | BID              | 100        | -55    | submission |
+      | lp1 | party0 | ETH/DEC19 | 100000            | 0.001 | buy  | BID              | 100        | 55     | amendment  |
```

Comment:
This makes no sense, both LPs share the same ID column, but are different parties, and the second one is an amendment?? I don't get what's going on here.

The fact that the parties are different makes even less sense considering both submissions/amendments are incomplete, only accounting for one side (sell/buy)

Reply:
agree, updated the LPs to the same party now